### PR TITLE
{Github} Update PR template

### PR DESCRIPTION
**Pull request by qwordy**
**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -6,6 +6,12 @@
 [Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
 [Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.
 
+**Description of PR (Mandatory)**  
+(Why this PR? What is changed? What is the effect? Etc. A high-quality description can accelerate the review process)
+
+**Testing Guide**  
+(Example commands with explanation)
```
Comment (jiasli) [inline .github/pull_request_template.md]:
```suggestion
(Example commands with explanations)
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -6,6 +6,12 @@
 [Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
 [Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.
 
+**Description of PR (Mandatory)**  
+(Why this PR? What is changed? What is the effect? Etc. A high-quality description can accelerate the review process)
```
Comment (jiasli) [inline .github/pull_request_template.md]:
```suggestion
(Why this PR? What is changed? What is the effect? etc. A high-quality description can accelerate the review process)
```

**Comment by qwordy** [comment]
Suggested change is a great function!

**Comment by fengzhou-msft** [comment]
Should we move the Description of PR and Testing Guide above History Notes as they are more important information for reviewers and Description is mandatory while History Notes is optional?

**Pull request merged**